hw/ip/entropy_src/doc/programmers_guide.md

```diff
@@ -92,6 +92,24 @@ Note that once the field `ES_ROUTE` is set to `kMultiBitBool4True`, no entropy i
 This mode is compatible with [Hardware Conditioning Bypass](#hardware-conditioning-bypass) (enabled by setting the `ES_TYPE` field in [`ENTROPY_CONTROL`](registers.md#entropy_control) to `kMultiBitBool4True`).
 
 
+## Single RNG Bit Mode
+
+The PTRNG entropy source delivers a stream of symbols to ENTROPY_SRC.
+Each of these symbols contains multiple bits.
+Firmware can determine whether ENTROPY_SRC should utilize all or only one of the bits that make up a symbol.
+Per default, the `RNG_BIT_ENABLE` field in the [`CONF`](registers.md#conf) register is set to `kMultiBitBool4False` and the whole symbol is used.
+To enable the single RNG bit mode `RNG_BIT_ENABLE` has to be set to `kMultiBitBool4True`.
+To select which specific bit should be used, the `RNG_BIT_SEL` field in the [`CONF`](registers.md#conf) register must be set to the number of the desired lane.
+When ENTROPY_SRC is in single RNG bit mode, only a subset of the health tests continue to be applicable.
+For this reason the thresholds, for both the bypass and the FIPS values, must be set accordingly.
+The repetition count test, markov test and adaptive proportion test can all still be performed on a single lane.
+However, to get the same amount of entropy we now must collect four times as many individual samples from PTRNG.
```

Comment:
However, to get the same number of entropy bits, we now must collect four times as many individual symbols from the PTRNG.